debug.log

```diff
@@ -0,0 +1,2 @@
+[1205/182024.817:ERROR:directory_reader_win.cc(43)] FindFirstFile: El sistema no puede encontrar la ruta especificada. (0x3)
```

Comment:
No need to commit this kind of files ;)

Reply:
This file was added to gitignore and deleted from git.